test/functional/tests/io_class/test_io_class_occupancy_resize.py

```diff
@@ -20,8 +20,8 @@
 
 @pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
 @pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
-@pytest.mark.parametrizex("cache_line_size", CacheLineSize)
-@pytest.mark.parametrize("new_occupancy", [0, 20, 70, 100])
+@pytest.mark.parametrizex("cache_line_size", [CacheLineSize])
```

Comment:
I need explanation - why did you put cache line size enum into list?

Reply:
It was left here accidentally, I removed it and rebased the PR